library/item/item_setting/src/main/java/org/quiltmc/qsl/item/setting/api/QuiltItemSettings.java

```diff
@@ -190,6 +183,7 @@ public <T> QuiltItemSettings customSetting(CustomItemSetting<T> setting, T value
 	}
 
 	// Overrides of vanilla methods
+	// TODO override remaining methods once they're mapped
```

Comment:
I think it would be better to make this an injected interface, then we dont have to worry about forgetting to make overrides for the different settings.